tests/main.tf

```diff
@@ -0,0 +1,127 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "lx-instance-centos6" {
+  source = "git::https://github.com/plus3it/terraform-aws-watchmaker//modules/lx-instance/"
```

Comment:
Sources need to be relative paths, to ensure the code in the pr is what is tested.

Reply:
derp.. updated